src/components/theme/types.ts

```diff
@@ -8,4 +8,8 @@ export interface ThemeContextProps {
     theme: Theme;
     themeValue: RealTheme;
     direction: Direction;
+    /**
+     * Ability to define soft migration for new list item view
```

Comment:
Let's clarify that the property is not for public usage

```suggestion
     * DO NOT USE. Internally used for soft migration to the new list item view
```